cmake/f3dPlugin.cmake

```diff
@@ -191,9 +203,17 @@ macro(f3d_plugin_build)
     CXX_VISIBILITY_PRESET hidden
     )
 
+  if(DEFINED f3d_INCLUDE_DIR)
+    # External plugin path
+    set(_f3d_include_path "${f3d_INCLUDE_DIR}/f3d")
```

Comment:
shouldnt f3d_INCLUDE_DIR already point to /f3d ?

Reply:
No, for a regular installation, it's just `/usr/include` so the user can use `#include <f3d/engine.h>`.  
For the plugins, we cannot use the `f3d/` prefix because headers are located in `library/public` directly.

Reply:
ok